packages/forms/src/UIForm/UIForm.container.js

```diff
@@ -151,9 +151,11 @@ export default class UIForm extends React.Component {
 				}
 				this.setErrors(event, errors);
 			}
-			if (typeof data.properties === 'function') {
+			if (data.properties) {
 				let properties = data.properties;
-				properties = data.properties(liveState.properties);
+				if (typeof data.properties === 'function') {
+					properties = data.properties(liveState.properties);
+				}
 				this.onChange(event, { properties });
```

Comment:
this cause TUI-350. Please update this onChange call

Reply:
As discussed together, this will come in another PR that doesn't introduce a breaking change, so we don't wait for 3.0 to have that

Reply:
#2041